### DISCUSSION TITLE
程序打包问题

### DISCUSSION THREAD

**Discussion by oneMuggle** [tonquer/picacg-qt#299] [Q&A]
请问是用的什么方法打包的呢，我使用pyinstaller打包pyqt程序，打包出来的文件夹会非常的大，我查阅了一下网上的资料，也没有发现比较好的解决方法

**Comment by tonquer** [comment]
多大

**Reply by oneMuggle** [reply]
感谢回复，我试了一下打包本项目，与您所打包的版本大小基本相当，我的另一项目打包出来大概800M，应该是使用numpy库从而引用了mkl库所导致的

**Reply by Z-fly** [reply]
你可以试试用nuitka打包
